packages/appeal-reply-service-api/package.json

```diff
@@ -20,6 +20,7 @@
   },
   "dependencies": {
     "@pins/common": "file:../common",
+    "axios": "^0.21.1",
```

Comment:
Across other systems we have used node-fetch rather than axios. Not a major thing but I usually like to be consistent with which library we use.

Reply:
Ah ok i wasn't sure as its also in the horizon services and axios makes it easier than node-fetch. Do you think its worth changing it out? We shouldnt be just sticking random HTTP calls in methods anyway but  we dont have a better method right now.